scripts/compile.ts

```diff
@@ -25,7 +25,7 @@ export async function compile (): Promise<void> {
 try {
   await compile()
 } catch (e) {
-  console.error('caught:', e.message)
+  console.error('caught:', e instanceof Error ? e.message : String(e))
```

Comment:
If the linter is complaining about this, another simpler and faster version would be something along these lines, which avoids the slower `instanceof`:

```js
e?.message || e
```

Reply:
However, `e?.message || e` isn't type safe. However, I'd just log `e` which is more useful and avoids the type issue.